### PR TITLE
Document valid pattern for floating tags

### PR DESCRIPTION
**Pull request by gsaslis**
I came across this issue when trying to use a tag that contains a `-` in an OpenShift Build Service (OSBS) build for 3scale API Management.

### CODE REVIEW

File: docs/users.rst
Comment (tkdchen) [inline docs/users.rst:1206]: A concrete example is always better than giving such a regular expression in documentation. We can't assume a user is familiar or even master the regular expression, and this regex is not much straightforward for a user to have a quick understand what a floating tag could look like. The unique tag example mentioned above is a good example to follow I think.

Comment (gsaslis) [reply docs/users.rst:1206]: While I don't think this should block merging this PR (as it already improves the situation - where the docs provide neither a concrete example, nor the validation rule), I'll go ahead and add one. ;)

Comment (gsaslis) [reply docs/users.rst:1207]: @tkdchen resolving this after adding an example, please reopen if you need something further.

**Comment by tkdchen** [comment]
@gsaslis You have to sign-off your commits.

**Comment by MartinBasti** [comment]
Any update on this PR?

**Comment by gsaslis** [comment]
@tkdchen / @MartinBasti please take a look.

**Review by MartinBasti — APPROVED** [review]

**Review by ben-alkov — APPROVED** [review]

**Pull request merged**
